### PR TITLE
Add decomp status

### PR DESCRIPTION
**Pull request by 0x32767**
I have added a script that shows the decomp progress. The script will also create a `progress.svg` file in the `resources` dir. The svg is updated if the `update_svg` param is passed to the cli.

### CODE REVIEW

File: resources/progress_template.svg [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<svg height="175" width="330" xmlns="http://www.w3.org/2000/svg">
+  <rect x="0" y="0" height="175" width="330" fill="black" />
+  <rect x="0" y="0" height="174" width="329" fill="white" />
+
+  <text x="5" y="30" font-size="35">Decomp Progress</text>
+  <text x="50" y="64" font-size="16" fill="black" font-weight="bold">Functions implimented:</text>
```
Comment (roblabla) [inline resources/progress_template.svg:6]:
```suggestion
  <text x="50" y="64" font-size="16" fill="black" font-weight="bold">Functions implemented:</text>
```

File: resources/progress_template.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  <rect width="322" height="28" x="4" y="80" fill="grey"/>
+  <rect width="{FUNC_PROG_WIDTH}" height="26" x="5" y="81" fill="white"/>
+  <text x="155" y="100" font-size="16" fill="black" font-weight="bold">{FUNC_PROG_PERCENT}%</text>
+
+  <text x="50" y="128" font-size="16" fill="black" font-weight="bold">Bytes implimented:</text>
```
Comment (roblabla) [inline resources/progress_template.svg:11]:
```suggestion
  <text x="50" y="128" font-size="16" fill="black" font-weight="bold">Bytes implemented:</text>
```

**Comment by wearrrrr** [comment]
LGTM! At some point I might go in and make the svgs look better :p, but so far they functionally work well.

**Comment by roblabla** [comment]
Done in #75 . Thank you for your contribution!

**Pull request closed without merging**